public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ResourcePicker/ResourcePicker.tsx

```diff
@@ -150,33 +138,48 @@ const ResourcePicker = ({
       ) : (
         <>
           <NestedResourceTable
-            rows={rows}
+            rows={azureRows}
             requestNestedRows={requestNestedRows}
             onRowSelectedChange={handleSelectionChanged}
             selectedRows={selectedResourceRows}
           />
 
           <div className={styles.selectionFooter}>
-            {selectedResourceRows.length > 0 && (
-              <>
-                <Space v={2} />
-                <h5>Selection</h5>
-                <NestedResourceTable
-                  rows={selectedResourceRows}
-                  requestNestedRows={requestNestedRows}
-                  onRowSelectedChange={handleSelectionChanged}
-                  selectedRows={selectedResourceRows}
-                  noHeader={true}
-                />
-              </>
-            )}
+            <h5>
+              Selected Scope{' '}
+              <Tooltip
+                content={
+                  <>
+                    <p>
+                      Select a resource above or type in the{' '}
+                      <a
+                        href="https://docs.microsoft.com/en-us/azure/azure-monitor/logs/log-standard-columns#_resourceid"
+                        rel="noopener noreferrer"
+                        target="_blank"
+                      >
+                        resource id (also referred to as a resource uri)
+                      </a>{' '}
+                      below. Grafana supports template variable interpolation like so:
+                    </p>
+                    <p>/subscriptions/$subId</p>
```

Comment:
What about "You can manually edit the [resource ID] or use template variables" (shorter but explaining everything?). If we need more info, we can add that to the docs and add a link here.